src/PerformanceTests/Tests/Categories/Base.cs

```diff
@@ -133,14 +133,4 @@ static void DeleteOtherPlatformHost(Permutation permutation)
             }
         }
     }
-
-    [SetUpFixture]
-    public class TestSessionInitializer
-    {
-        [SetUp]
```

Comment:
Should this not be `[OneTimeSetup]` instead of replacing it with a static initialization?

I mean, the static init works fine, so I guess this is just a style question.